Resources/Prototypes/Traits/species.yml

```
@@ -66,6 +68,33 @@
         - Swashbuckler
         - Spearmaster
+        - Mystic
+
+- type: trait
+  id: Mystic
+  category: Mental
+  points: -1
+  functions:
+    - !type:TraitReplaceComponent
+      components:
+      - type: OniDamageModifier
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)